### ISSUE TITLE
Loss cannot converge

### ISSUE THREAD

**Issue by icicle4** [ryanaleksander/softened-similarity-learning#9]
Thanks for your repo. I did not use the Memory Table file you provided but used ImageNet pre-trained resnet50 to directly build the Memory Table. Under the conditions of BatchSize = 32, epoch = 30, and LR = 0.1, the loss can no longer decrease when it reaches 60. What special optimization techniques did you use to reduce the loss correctly?

**Comment by ryanaleksander** [comment]
Thank you for checking out the repo. The default config is the config that I've been using. I think your learning rate is a bit too large, maybe you can try 0.01 and see if it helps.

**Comment by icicle4** [comment]
Thank you for your reply.

I have tried a lot of learning rates, and it cannot solve the convergence problem.

In addition, the difference between the settings in my repo and your repo is that because I feel that your image input is too large, and the input size of the image is not mentioned in the original paper, the input size of my image is 256x128. Is it because the input size is too small?  Or can you tell me why you give such a large image input size?

**Comment by ryanaleksander** [comment]
I'm sorry but it's been a while so I don't remember the exact reason. I probably just picked 256x256 as the starting size, then increased the height to make it easier to  compute part similarities.

**Comment by icicle4** [comment]
thanks
